src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3766,6 +3766,11 @@ public override SyntaxNode DefaultExpression(ITypeSymbol type)
             return DefaultExpression(type.GenerateTypeSyntax());
         }
 
+        internal override SyntaxNode AddParentheses(SyntaxNode expression)
+        {
+            return Parenthesize(expression);
+        }
+
         private static ExpressionSyntax Parenthesize(SyntaxNode expression)
```

Comment:
The reason I used two names is because this existing method returns an `ExpressionSyntax`, but the new method returns a `SyntaxNode`. I didn't want to modify/burden all the existing callsites with casts.

Reply:
Ah, gotcha!  Makes sense.